hibernate-core/src/main/java/org/hibernate/tool/schema/internal/exec/ScriptSourceInputAggregate.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.tool.schema.internal.exec;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.hibernate.tool.hbm2ddl.ImportSqlCommandExtractor;
+import org.hibernate.tool.schema.spi.ScriptSourceInput;
+
+/**
+ * A script source input that aggregates over multiple other {@link ScriptSourceInput}.
+ *
+ * @author Christian Beikov
+ */
+public class ScriptSourceInputAggregate implements ScriptSourceInput {
```

Comment:
My comment seems frivolous, but I checked existing name convention for such similar stuff and found we usually adopted 'Aggregatedxxxxx' or 'Aggregatexxxx'.
Feel free to ignore my comment though.